src/components/calculator/RadioControl.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Badge, Form, OverlayTrigger } from 'react-bootstrap'
+
+import { CheckBoxFormValue } from 'data/data'
+
+export const RadioControl: React.FunctionComponent<{
+  id: string
+  setter: (value: string) => void
+  source: Array<CheckBoxFormValue>
+  value: string | number
+  label?: string
+  header?: string
+  helpText?: string
+  popover?: JSX.Element
+  className?: string
+}> = (props) => (
+  <Form.Group controlId={props.id}>
+    {(props.label || props.header) && (
```

Comment:
Hm, It'd be nice to refactor this label stuff that's shared between RadioControl and FormControl. Maybe in a follow-up.

Reply:
Done. :)